src/deployment/data_migration.py

```diff
@@ -38,8 +42,38 @@ def migrate_task_os(table_service: TableService) -> None:
     print("migrated %s rows" % count)
 
 
+def migrate_notification_keys(table_service: TableService) -> None:
+    table_name = "Notification"
+    notifications = table_service.query_entities(
+        table_name, select="PartitionKey,RowKey,config"
+    )
+    partitionKey = None
+
+    count = 0
+    for entry in notifications:
+        try:
```

Comment:
Could you add a comment about why this a non-exceptional UUID parse means we don't have to do anything?